src/engine/strat_engine/thinpool/thinpool.rs

```diff
@@ -438,9 +451,11 @@ impl ThinPool {
                             extend_size: MetaBlocks,
                             backstore: &mut Backstore)
                             -> EngineResult<MetaBlocks> {
+        let backstore_device = self.backstore_device;
```

Comment:
Forward-looking. Is correct within current context, but would seem unduly paranoid w/out expected changes.